classifier_update params
---
bool result
